### PR TITLE
chore: make windows bin md5 sum file name match others

### PR DESCRIPTION
**Pull request by dannyrandall**
All of the other md5 sums of our output resources are named `filename.md5` except for windows, which has `.exe` stripped off the filename. We don't strip off the `.exe` for the signature files (`.asc`) on windows, so this makes the windows md5 sum match the windows signature file and the rest (linux & mac) of the resources.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: .release/buildspec_stage.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ phases:
       - |
         for artifact in `cat ./$GIT_COMMIT_ID.manifest`
         do
-            md5FileName="${artifact%.exe}"
```
Comment (efekarakus) [inline .release/buildspec_stage.yml:25]: im so confused - we used to have ".exe" at the end of all md5 files? but that's clearly not the case here: https://github.com/aws/copilot-cli/releases what am I misunderstanding?

Comment (efekarakus) [inline .release/buildspec_stage.yml:25]: im so confused - we used to have ".exe" at the end of all md5 files? but that's clearly not the case here: https://github.com/aws/copilot-cli/releases what am I misunderstanding?

Comment (dannyrandall) [reply .release/buildspec_stage.yml:25]: so for some reason on the windows file, we trim off the `.exe`, but only for the md5 - in the `.asc` file we leave it on. i just want them to be the same :sob:. 

so today our outputs are:
```
copilot-windows.exe
copilot-windows.exe.asc
copilot-windows.md5
```

after this change (theoretically):
```
copilot-windows.exe
copilot-windows.exe.asc
copilot-windows.exe.md5
```

Comment (efekarakus) [reply .release/buildspec_stage.yml:25]: ok my bad! I didn't realize "%.exe" removed ".exe" 😂 now evreything makes sense

Comment (efekarakus) [reply .release/buildspec_stage.yml:25]: ok my bad! I didn't realize "%.exe" removed ".exe" 😂 now evreything makes sense

**Review by efekarakus — APPROVED** [review]

**Review by huanjani — APPROVED** [review]

**Review by iamhopaul123 — APPROVED** [review]

**Pull request merged**